go.mod

```diff
@@ -4,20 +4,28 @@ go 1.13
 
 require (
 	github.com/10gen/escaper v0.0.0-20160802155924-17fe61c658dc
-	github.com/10gen/llmgo v0.0.0-20180924181635-afcced55e92d
 	github.com/10gen/openssl v0.0.0-20190731121843-f50700cbcaf4
-	github.com/golang/snappy v0.0.1 // indirect
+	github.com/golang/snappy v0.0.1
 	github.com/google/gopacket v1.1.17
 	github.com/howeyc/gopass v0.0.0-20190910152052-7cb4b85ec19c
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/kr/pretty v0.2.0 // indirect
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/smartystreets/goconvey v1.6.4
 	github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572 // indirect
-	github.com/xdg/stringprep v1.0.0 // indirect
+	github.com/xdg/stringprep v1.0.0
 	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
 	golang.org/x/text v0.3.2 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
-	gopkg.in/yaml.v2 v2.2.8 // indirect
+	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
+	gopkg.in/yaml.v2 v2.2.8
 )
+
+replace github.com/10gen/mgo => github.com/10gen/mgo v0.0.0-20181212170345-8c133fd1d0fc
```

Comment:
Should this be `replace gopkg.in/mgo.v2 => github.com/10gen/mgo v0.0.0-20181212170345-8c133fd1d0fc`?

Similar question for other replace directives.

Reply:
Thanks for catching this! I don't find other replace directives with the `.v2` issue though since they don't have the version inside their module name?

Reply:
Ah, I got what you mean! I made the change to the module's domain in `mgo` and `text`, used the replace section as well.